android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java

```diff
@@ -183,7 +187,16 @@ public void setLargeIcon(Object icon)
 		}
 		setProperty(TiC.PROPERTY_LARGE_ICON, icon);
 	}
-
+    
+	@Kroll.method @Kroll.setProperty
+	public void setColor(String color)
+	{
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)) {
```

Comment:
Please, import `android.os.Build`
One extra bracket is added after the `if` statement.

Reply:
Done